index.js

```diff
@@ -0,0 +1,19 @@
+const fetchPeeps = require('./fetchPeeps')
+const newUser = require('./createUser')
+const newSession = require('./createSession')
+
+const button = document.querySelector('#signup-button')
+
+button.addEventListener('click', () => {
+  handle = document.querySelector('#username').value;
+  password = document.querySelector('#password').value;
+  user = {"user": { "handle": `${handle}`, "password": `${password}`}};
+  newUser(user);
```

Comment:
Maybe pass handle and password to the newUser function as two arguments, rather than grouping them into one?